### ISSUE TITLE
Command IDs are sometimes duplicated

### ISSUE THREAD

**Issue by DestyKarpov** [gigobyte/HLTV#37]
I use [getMatches](https://github.com/gigobyte/HLTV#getmatches) method in my project. Most recently, I started to notice that some games have identical IDs of two commands. I think you should check this out. Thank you!

**Comment by gigobyte** [comment]
Can you please share some match IDs that have wrong teams, it's very hard to debug this issue otherwise.

**Comment by gigobyte** [comment]
```
{ id: 2318403,
    date: 1515351600000,
    team1: { name: 'GODSENT', id: 6902 },
    team2: { name: 'eXtatus', id: 6745 },
    format: 'bo3',
    map: undefined,
    title: '',
    event: { name: 'ESL Pro League Season 6 Europe Relegation', id: 3359 },
    stars: 0,
    live: false }
```

The first match is currently alright

**Comment by DestyKarpov** [comment]
![2018-01-09_01-25-34](https://user-images.githubusercontent.com/24729953/34683115-09380220-f4dc-11e7-9201-9f02d92027de.png)

**Comment by DestyKarpov** [comment]
![2018-01-09_02-28-53](https://user-images.githubusercontent.com/24729953/34685966-030a7032-f4e5-11e7-9b10-59cda0dc2adf.png)
I think this is due to the fact that certain games do not always display team logos. But I can be wrong.

**Comment by gigobyte** [comment]
Thanks, noted! What did you use for the arrows on the screenshots though, looks nice :smile:

**Comment by DestyKarpov** [comment]
Yes, I also like it, because it's very convenient and fast!  I used Yandex.Disk, here's the link: [https://disk.yandex.ru/screenshot/](https://disk.yandex.ru/screenshot/)

**Comment by gigobyte** [comment]
I localized the issue: http://prntscr.com/hzfble
There will be a fix in the coming days.

**Comment by DestyKarpov** [comment]
Excellent! Thanks you!
